docs/config/lua/config/font_rules.md

```diff
@@ -45,46 +48,72 @@ return {
 }
 ```
 
-Here's an example from my configuration file;
+Here's an example combining Fira Code and Victor Mono;
 
 ```lua
 local wezterm = require 'wezterm'
-
--- A helper function for my fallback fonts
-function font_with_fallback(name, params)
-  local names = { name, 'Noto Color Emoji', 'JetBrains Mono' }
-  return wezterm.font_with_fallback(names, params)
-end
-
 return {
-  font = font_with_fallback 'Operator Mono SSm Lig Medium',
+  font = wezterm.font_with_fallback {
+    { family = 'FiraCode', weight = 'Light' },
+    'Terminus',
+  },
+  harfbuzz_features = {
+    'cv13',
```

Comment:
It would be good to explain what these do in a comment, as this is the kind of thing that looks mysterious and folks will just copy because they're not sure if they need it or not, and then it will end up get copied around and used in situations where it doesn't make sense.

Alternatively, moving this into line 57 above to make it actually apply only to FiraCode would help to reinforce that it belongs with that font:

```lua
   { family = 'FiraCode', weight = 'Light',
        -- adjust presentation of FiraCode
        -- see: https://github.com/tonsky/FiraCode/wiki/How-to-enable-stylistic-sets
        harfbuzz_features= {
        'cv13',
```

Reply:
Pushed changes with more comments addressing this, your suggestion actually doesn't apply to bold typeface Fira Code in that case, due to weight = 'Light'.
           Made that clear as well, and have checked that this indeed does work as expected, and in ways promotes a desire to tweak the weightings, and perhaps complete the micromanagement for other things like underline and strikethrough, but think that is going way too far with the example.

And leaving Terminus fallback with comments for coherency and also showing another way to set font.

Reply:
There is another thing,   'ss04' changes zero for Victor Mono, and changes $ for Fira Code.

Reply:
https://www.codingfont.com/blog/coding-font-with-italic-and-programming-ligature/

According to here, it covers two out of three open fonts with ligatures, and one of them being one of three best with true italics or cursive italics. So going over board here should be a mute point, as many may find wezterm for naming them in an example.

Reply:
Added comment on ss04 and link to Victor.
If weight up slightly, looks like this.
https://ibb.co/DMnggpQ

Reply:
Hmm, I think that we should take the `harfbuzz_features` part out of this example.  The rationale is: it is insufficiently explained here, and too complicated to explain here without dominating what would otherwise be a pretty straightforward example of using `font_rules`.

Rather than draw questions or feeling that `font_rules` are too complex, I think dropping the stylistic sets from this initial example makes sense.